website/docs/trigger.md

```diff
@@ -209,7 +206,7 @@ settings:
 * `name` is set to the `triggername`
 * `command` is set to the `<cmd>` list
 * `expression` is generated from the `<patterns>` list using the rules laid
-  out in [Simple Pattern Syntax](/watchman/docs/simple-query.html)
+  out in [Simple Pattern Syntax](/docs/simple-query)
```

Comment:
A lot of these links have been changed from `/watchman/docs/` to just `/docs/`.  I think that makes sense for local testing, but when these are deployed to github that isn't right (eg: https://facebook.github.io/watchman/docs/install.html is where the install docs get served up).

Can these be changed to omit the paths?  For example could this one could perhaps just be `simple-query` instead of `/docs/simple-query`?

Reply:
I tested, isn't works, because the `/`, serves to identify that after the base url, in this case `facebook.github.io`, the route will be `docs/simple-query` if we change this, the docusaurus will cannot understand the path

Reply:
Yeah, sorry again I didn't tested using this way. I'll change ASAP.

Reply:
Thanks; I think the leading `/` might not be good because the docs run out from `/watchman/` and `/` will be the wrong directory.  Is there a way to test with a subdir locally?

Reply:
Yes, I do thinks. Good observation.